charts/tekton-chains/templates/Deployment-tekton-chains-controller.yml

```diff
@@ -59,3 +67,8 @@ spec:
                   path: oidc-token
                   expirationSeconds: 600 # Use as short-lived as possible.
                   audience: sigstore
+        {{- if .Values.spire.enabled }}
+        - name: spiffe-workload-api
+          hostPath:
+            path: {{ .Values.spire.socketPath }}
+        {{- end }}
```

Comment:
Can you a new line here. I think the linter will fail if not.

Reply:
👍🏽